Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -2324,6 +2327,107 @@ auto B3IRGenerator::addI31GetU(ExpressionType ref, ExpressionType& result) -> Pa
     return { };
 }
 
+auto B3IRGenerator::addStructNew(uint32_t typeIndex, Vector<ExpressionType>& args, ExpressionType& result) -> PartialResult
+{
+    const auto type = Type { TypeKind::Ref, Nullable::No, m_info.typeSignatures[typeIndex]->index() };
+    PatchpointValue* patch = m_proc.add<PatchpointValue>(toB3Type(type), origin());
+    patch->append(instanceValue(), ValueRep::reg(GPRInfo::argumentGPR0));
```

Comment:
Missing FP?

Reply:
sorry, don't understand why should I add floating point reg here, could you please clarify it a bit?

Reply:
This patchpoint expects the call frame as the second argument, right?

Reply:
right, I've added it